packages/babel-plugin-proposal-decorators/test/fixtures/2023-11-classes/replacement-static-private-environment-super/exec.js

```diff
@@ -0,0 +1,38 @@
+{
+  "different private/super access in a static private method";
+
+  const dummy = () => {}
+
+  let b;
+
+  const idFactory = (hint) => {
+    return class { static ["id" + hint](v) { return v } }
+  }
+
+  class C extends idFactory("C") { #x = "C#x"; }
+
+  const dec = (b) => {
+    originalB = b;
+    return C;
+  }
+
+  class A extends idFactory("A") {
+    #x = "A#x";
+    static *[Symbol.iterator]() {
+      @dec
+      class B extends idFactory("B") {
+        #x = "B#x";
+        @(yield super.idA(o => o.#x), dummy)
```

Comment:
Currently the Node.js 16 CI is failing but Node.js > 18 is fine, I guess it was an engine bug, I will investigate.

Reply:
Interesting.

According to CI, this test is failing on Node.js 16 + *nix, but it works on Node.js 16 + Windows.
If I only run this test on Node.js + macOS, it works fine.

So it points to an OS-specific optimizer bug. I will mark this test as node.js 18+.